DS and Algorithm/Selenium/SeleniumExample.java

```diff
@@ -0,0 +1,23 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/6#issuecomment-2168361604>#a9a198</a><span> - 06/14/2024, 04:26 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The WebDriver instance is initialized without any exception handling. If the WebDriver initialization fails, it will cause the application to crash.
</div>
<div id="fix">
<b>Fix:</b> Add exception handling around the WebDriver initialization to ensure the application can handle initialization failures gracefully.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -4,7 +4,12 @@
      // Initialize WebDriver
 -    WebDriver driver = WebDriverSetup.initializeChromeDriver();
 +    WebDriver driver;
 +    try {
 +        driver = WebDriverSetup.initializeChromeDriver();
 +    } catch (Exception e) {
 +        e.printStackTrace();
 +        return;
 +    }
```
</div>
</details>
</div>
<br>


</div>